subsys/net/lib/sntp/sntp_pkt.h

```diff
@@ -9,21 +9,21 @@
 
 #include <zephyr/types.h>
 
-#define LI_MASK   0xC0
-#define VN_MASK   0x38
-#define MODE_MASK 0x07
+#define Z_SNTP_LI_MASK   0xC0
+#define Z_SNTP_VN_MASK   0x38
+#define Z_SNTP_MODE_MASK 0x07
 
```

Comment:
Could you remove the SNTP naming changes from this PR?

Reply:
> Could you remove the SNTP naming changes from this PR?

Can you please approve https://github.com/zephyrproject-rtos/zephyr/pull/24756, and ideally merge it, then I'll rebase this PR on master.